sci-geosciences/grass/grass-7.8.2.ebuild

```diff
@@ -0,0 +1,279 @@
+		$(use_with opengl)
+		$(use_with odbc)
+		$(use_with fftw)
+		$(use_with blas)
+		$(use_with lapack)
+		$(use_with X cairo)
+		$(use_with truetype freetype)
+		$(use_with truetype freetype-includes "${EPREFIX}/usr/include/freetype2")
+		$(use_with nls)
+		$(use_with readline)
+		$(use_with threads pthread)
+		$(use_with openmp)
+		$(use_with opencl)
+		$(use_with liblas liblas "${EPREFIX}/usr/bin/liblas-config")
+		$(use_with X wxwidgets "${WX_CONFIG}")
+		$(use_with netcdf netcdf "${EPREFIX}/usr/bin/nc-config")
+		$(use_with geos geos "${EPREFIX}/usr/bin/geos-config")
+		$(use_with X x)
+	)
+	econf "${myeconfargs[@]}"
+}
+
+src_compile() {
+	# we don't want to link against embedded mysql lib
+	emake CC="$(tc-getCC)" MYSQLDLIB=""
+}
+
+src_install() {
+	emake DESTDIR="${D}" \
+		INST_DIR="${D}/usr/$(get_libdir)/${MY_PM}" \
+		prefix="${D}/usr/" BINDIR="${D}/usr/bin" \
+		PREFIX="${D}/usr/" \
+		install
+
+	pushd "${D}/usr/$(get_libdir)/${MY_PM}" &> /dev/null || die
+
+	local HTML_DOCS=( docs/html/. )
+	einstalldocs
+
+	# manuals
+	dodir /usr/share/man/man1
+	rm -rf man/ || die
+
+	# translations
+	if use nls; then
+		dodir /usr/share/locale/
+		mv locale/* "${D}/usr/share/locale/" || die
+	fi
+
+	popd &> /dev/null || die
+
+	# link libraries in the ~standard~ place
+	for fLib in $(ls "${D}/usr/$(get_libdir)/${MY_PM}/lib/"); do
+		dosym "${MY_PM}/lib/${fLib}" "/usr/$(get_libdir)/${fLib}"
+	done
+
+	# link headers in the ~standard~ place
+	dodir "/usr/include/"
+	dosym "../$(get_libdir)/${MY_PM}/include/grass" "/usr/include/grass"
+
+	# fix paths in addons makefile includes
+	local scriptMakeDir
+	scriptMakeDir="${D}/usr/$(get_libdir)/${MY_PM}/include/Make/"
+	for mkFile in $(ls "${scriptMakeDir}"); do
+		echo sed -i "s|${D}|/|g" "${scriptMakeDir}/${mkFile}" || die
+		sed -i "s|${D}|/|g" "${scriptMakeDir}/${mkFile}" || die
+	done
+
+	# get proper folder for grass path in script
+	local gisbase
+	gisbase="${ROOT}/usr/$(get_libdir)/${MY_PM}"
+	sed -e "s:gisbase = \".*:gisbase = \"${gisbase}\":" \
+		-i "${D}/usr/bin/${MY_PM}" || die
+
+	# get proper fonts path for fontcap
+	sed -i \
+		-e "s|${D}/usr/${MY_PM}|${EPREFIX}/usr/$(get_libdir)/${MY_PM}|" \
+		"${D}/usr/$(get_libdir)/${MY_PM}/etc/fontcap" || die
+
+	# set proper python interpreter
+	sed -e "s:= \"python\":= \"${EPYTHON}\":" -i "${D}/usr/bin/${MY_PM}" || die
+
+	if use X; then
+		local GUI="-gui"
+		[[ ${WX_BUILD} == yes ]] && GUI="-wxpython"
+		make_desktop_entry "/usr/bin/${MY_PM} ${GUI}" "${PN}" "${PN}-48x48" "Science;Education"
+		doicon -s 48 gui/icons/${PN}-48x48.png
+	fi
+
+	# install .pc file so other apps know where to look for grass
+	insinto /usr/$(get_libdir)/pkgconfig/
+	doins grass.pc
+
+	# fix weird +x on tcl scripts
+	find "${D}" -name "*.tcl" -exec chmod +r-x '{}' \;
+}
+
+pkg_postinst() {
+	use X && xdg_pkg_postinst
+
+	ewarn "GRASS addons may fail due to Python 3 incompatibility."
+	ewarn "If that is tha case you can change the shebang a the beginning of"
```

Comment:
A couple of small typos here - not yours, but an opportunity to fix nonetheless.